recipe/meta.yaml

```diff
@@ -54,34 +80,41 @@ outputs:
     requirements:
       host:
         - sysroot_{{ ctng_target_platform }} {{ conda_glibc_ver }}
-        - {{ pin_subpackage("libgomp", exact=True) }}
-        - {{ pin_subpackage("libstdcxx-ng", exact=True) }}
-        - {{ pin_subpackage("libgcc-ng", exact=True) }}
-        - {{ pin_subpackage("libgfortran-ng") }}
+        - {{ pin_subpackage("libgomp", exact=True) }}           # [target_platform == ctng_target_platform]
+        - {{ pin_subpackage("libstdcxx-ng", exact=True) }}      # [target_platform == ctng_target_platform]
+        - {{ pin_subpackage("libgcc-ng", exact=True) }}         # [target_platform == ctng_target_platform]
+        - {{ pin_subpackage("libgfortran-ng") }}                # [target_platform == ctng_target_platform]
       run:
         - {{ pin_subpackage("binutils_impl_" ~ ctng_target_platform, max_pin=None) }}
+        - {{ pin_subpackage("libgcc-devel_" ~ ctng_target_platform, exact=True) }}
         # libstdcxx-ng is a runtime dep of gcc because LTO requires it.
-        - {{ pin_subpackage("libstdcxx-ng", max_pin=None) }}
-        - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
-        - {{ pin_subpackage("libgomp", max_pin=None) }}
+        - {{ pin_subpackage("libstdcxx-ng", max_pin=None) }}    # [target_platform == ctng_target_platform]
+        - {{ pin_subpackage("libgcc-ng", max_pin=None) }}       # [target_platform == ctng_target_platform]
+        - {{ pin_subpackage("libgomp", max_pin=None) }}         # [target_platform == ctng_target_platform]
         - sysroot_{{ ctng_target_platform }}
+        # Any version newer than the devtoolset works. This can be removed when bootstrapping.
```

Comment:
A few questions here:

1. Why are the requirements different for native versus cross compilers here?

2. Why are we not adding a requirement on libgomp for cross compilers?

Reply:
See my previous answer